src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/DomainEventServiceTest.kt

```diff
@@ -34,6 +37,8 @@ import java.util.UUID
 class DomainEventServiceTest {
   private val domainEventRepositoryMock = mockk<DomainEventRepository>()
   private val hmppsQueueServiceMock = mockk<HmppsQueueService>()
+  private val userService = mockk<UserService>()
+  private val user = UserEntityFactory().withDefaultProbationRegion().produce()
```

Comment:
hmm..I think for CAS2 it would be better to set the user id to `null`, or we need to implement our own function to find the user ID.
We have a different set of users on CAS2, they will be Nomis users or External users rather than Delius, so the current `getUserForRequestOrNull` used will always return null.